CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [4.3.0]
+### Changed
+- Add early return if JSON RPC request is not found ([#35](https://github.com/MetaMask/json-rpc-middleware-stream/pull/35))
```

Comment:
Hmm, this seems more like a bug fix to me, rather than a change.

Would you mind re-creating this release as v4.2.1? And making this a fix rather than a change.

The lockfile bumps can be omitted as well; they don't affect the package.